backend/build.gradle

```diff
@@ -34,6 +40,7 @@ dependencies {
     compile libs.spring_boot_security
     compile libs.spring_boot_jpa
     compile libs.database_hsqldb
+    compile "com.h2database:h2:${versions.h2}"
```

Comment:
better move this to the main build.gradle and create libs something like libs.database_h2 and replace with "compile libs.database_h2"